dkist/io/asdf/converters/file_manager.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+
+from asdf.extension import Converter
+
+
+class FileManagerConverter(Converter):
+    tags = [
+        "tag:dkist.nso.edu:dkist/array_container-0.2.0",
+        "asdf://dkist.nso.edu/array_container-1.0.0",
```

Comment:
If you like, you can register a YAML shorthand for your tag prefix:

https://github.com/asdf-format/asdf/blob/master/asdf/extension/_extension.py#L101

For example, if you use `!dkist!` as shorthand for `asdf://dkist.nso.edu/`, the ASDF file will be rendered like this:

```
!dkist!array_container-1.0.0
```

The only benefit is to reduce clutter when somebody pops open the file in an editor.

Reply:
Ohh, that's cool but I am not sure I really want to do it. I can't see many people opening the file and if they do the extra verbosity is probably helpful.